masonry/src/widgets/split.rs

```diff
@@ -143,17 +137,25 @@ impl<ChildA: Widget + ?Sized, ChildB: Widget + ?Sized> Split<ChildA, ChildB> {
 }
 
 // --- MARK: INTERNALS
+// TODO - Remove this block, and remove pixel-snapping code from this file.
+impl Length {
+    #[doc(hidden)]
+    pub fn ceil(self) -> Self {
+        Self::px(self.get().ceil())
+    }
+}
```

Comment:
Hmm, I'm not hugely happy with this - I'd probably prefer it to be written out manually, or as a `ceil_length`, say.

(I didn't particularly mind it in the length.rs file; obviously it's not great, but here is not good at all; non-local impls can be confusing).

Reply:
I'm putting it here because `split.rs` is the only file that uses that impl (that and the xilem `split.rs`).

I guess `ceil_length` works too.